### DISCUSSION TITLE
When sorting by artist, beatmaps aren't sorted alphabetically

### DISCUSSION THREAD

**Discussion by garaevdi** [ppy/osu#26308] [Q&A] [answered]
In stable, when sorting by artist, all beatmaps are sorted alphabetically, which helps with locating desired beatmap at a glance. Lazer just sorts them randomly.
| Stable | Lazer |
|--------|--------|
| ![screenshot010](https://github.com/ppy/osu/assets/64484944/54b71fd5-dbc5-41fa-9379-b5ae979de660) | ![image](https://github.com/ppy/osu/assets/64484944/cbb8d23a-b3b4-421d-a86f-5a308d343783) | 

In stable I don't have to search for another mapset with the same song, it's already near currently selected one.

(this probably should be an issue, but maybe I missed that one comment on some unrelated issue, which explains this behavior, so let it be here)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/27548
